index.js

```diff
@@ -38,23 +38,23 @@ const options = { sortBy, limit, reverse, minimal };
 	init(minimal);
 	const [input] = cli.input;
 	input === 'help' && (await cli.showHelp(0));
-	const states = input === 'states' ? true : false;
+	const states = input === 'states';
 	const country = input;
 
 	// Table
 	const head = xcolor ? single : colored;
 	const headStates = xcolor ? singleStates : coloredStates;
+
 	const border = minimal ? borderless : {};
-	const table = !states
-		? new Table({ head, style, chars: border })
-		: new Table({ head: headStates, style, chars: border });
+	const tableHead = states ? headStates : head;
+	const table = new Table({ head: tableHead, style, chars: border });
```

Comment:
logic for table was complex - it started with negation even though there was no real reason for it. As I needed this piece of code I refactored it a little bit for easier maintenance.